ckanext/datagovuk/lib/mailer.py

```diff
@@ -0,0 +1,13 @@
+from ckan.lib import mailer
+import ckan.model as model
+
+log = __import__('logging').getLogger(__name__)
+
+
+def send_password_alert(user):
+    try:
+        body = "Your password has been changed, if you haven't done it yourself, let us know"
+        mailer.mail_user(user, 'Your CKAN password has changed', body)
```

Comment:
Will users understand "CKAN"? Perhaps use "Your data.gov.uk publisher password..."?

Reply:
Password changes are on the CKAN publisher website, so I'm assuming that they know about it? Can easily change to `data.gov.uk publisher` though